From 4a89c8565d410930041edbdbd83a0004c7543e83 Mon Sep 17 00:00:00 2001
From: "Benjamin A. Beasley" <code@musicinmybrain.net>
Date: Sun, 10 Sep 2023 09:34:53 -0400
Subject: [PATCH] Remove useless shebang lines
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Since these files do not have the executable bit set in their file
permissions, the shebang line (#!) does nothing.

For qtsass/cli.py, the file is not even script-like (the “main routine”
is only defined, not called, on import).

For the other three, setting the executable bit *could* be an
alternative, if people wanted to run e.g. “./run_checks_and_format.py”
instead of “python3 run_checks_and_format.py”.
---
 qtsass/__main__.py       | 1 -
 qtsass/cli.py            | 1 -
 run_checks_and_format.py | 1 -
 setup.py                 | 1 -
 4 files changed, 4 deletions(-)

diff --git a/qtsass/__main__.py b/qtsass/__main__.py
index d4cd017..3d74b59 100644
--- a/qtsass/__main__.py
+++ b/qtsass/__main__.py
@@ -1,4 +1,3 @@
-#!/usr/bin/env python
 # -*- coding: utf-8 -*-
 # -----------------------------------------------------------------------------
 # Copyright (c) 2015 Yann Lanthony
diff --git a/qtsass/cli.py b/qtsass/cli.py
index 51ef727..1a46f7f 100644
--- a/qtsass/cli.py
+++ b/qtsass/cli.py
@@ -1,4 +1,3 @@
-#!/usr/bin/env python
 # -*- coding: utf-8 -*-
 # -----------------------------------------------------------------------------
 # Copyright (c) 2015 Yann Lanthony
diff --git a/run_checks_and_format.py b/run_checks_and_format.py
index 9adbf8e..8e18a6b 100644
--- a/run_checks_and_format.py
+++ b/run_checks_and_format.py
@@ -1,4 +1,3 @@
-#!/usr/bin/env python
 # -*- coding: utf-8 -*-
 # -----------------------------------------------------------------------------
 # Copyright (c) 2015 Yann Lanthony
diff --git a/setup.py b/setup.py
index df9c80f..77bbba7 100644
--- a/setup.py
+++ b/setup.py
@@ -1,4 +1,3 @@
-#!/usr/bin/env python
 # -*- coding: utf-8 -*-
 # -----------------------------------------------------------------------------
 # Copyright (c) 2015 Yann Lanthony
